tests/e2e/gc_block_volume_expansion.go

```diff
@@ -2000,6 +2001,158 @@ var _ = ginkgo.Describe("[csi-guest] Volume Expansion Test", func() {
 
 	})
 
+	/* Verify Online block volume expansion triggered when SVC CSI pod is down succeeds once SVC CSI pod comes up.
+	   Steps:
+	 	1. Create a SC with allowVolumeExpansion set to 'true' in GC.
+		2. create a PVC using the SC created in step 1 in GC and wait for binding
+		    with PV.
+		3. create a pod using the pvc created in step 2 in GC and wait FS init.
+	    4. bring CSI-controller pod down in SVC.
+	    5. Resize PVC with new size in GC.
+	    6. check for retries in GC.
+	    7. PVC in GC is in "Resizing" state and PVC in SVC has no state related
+		    to expansion.
+		8. bring the CSI-controller pod up in SVC.
+	    9. wait for PVC Status Condition changed to "FilesystemResizePending"
+	       in GC.
+	    10. Check Size from CNS query is same as what was used in step 7.
+	    11. wait for new size of PVC in GC and compare with SVC PVC size for
+	       equality.
+	    12. delete the pod, pvc,sc
+	*/
+	ginkgo.It("verify online block volume expansion triggered when SVC CSI pod is down succeeds once SVC CSI pod comes up", func() {
```

Comment:
This test should be marked destructive test so that the test runs before any other tests in e2e.

Please check with Chethan about the tag to use

Reply:
For now, We have the respective change to handle destructive test in case of block "csi-block-vanilla-destructive"
Same is not available for GC . We need to take that in separate PR. 
Till that I will make a TODO point in this test to add it for descrtictive test